packages/shared/util.js

```diff
@@ -2,7 +2,7 @@
 import Vue from 'vue'
 import semver from 'semver'
 import { VUE_VERSION } from './consts'
-import { config } from '@vue/test-utils'
+import { config } from '../test-utils/src'
```

Comment:
not sure it's ideal to import across packages but 🤷 seems to work great here.

I don't know why this project is a mono-repo.